index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body> 
+    <header id="header" class="header">
+        <img src="fila.png" alt="yeezy shoes" class="header__img" id="header-img">
+        <nav id="nav-bar" class="nav-bar">
+            <li>
+                <a href="#header" class="nav-link">Home</a>
+            </li>
+            <li>
+                <a href="#video-container" class="nav-link">Video</a>
+            </li>
+            <li>
+                <a href="#form" class="nav-link">Form</a>
+            </li>
```

Comment:
Remember to use the <ul> tag to contain the <li>, so that your code is more readable. 😄